dalite_xblock/public/js/dalite_xblock.js

```diff
@@ -4,8 +4,97 @@ function DaliteXBlock(runtime, element) {
     // hack to make LTI Consumer css applied to Dalite XBlock
     $(element).addClass("xblock-student_view-lti_consumer xblock-student_view");
     $(element).children(".xblock-dalite").addClass("lti_consumer");
-    
+
+    $(function ($) {
```

Comment:
It's bugging me that you had to add this much code just to trigger `refreshXBlock`.. especially since it really should just be an event that LtiConsumerXBlock iFrameModal triggers on close.

@bradenmacdonald could you advise?  The issue is that when the user edits a question in the Admin modal dialog, the updates don't get shown until you reload the Studio page.  What's the best path to take?

Note that the immediate client issue with seeing the Admin view in Studio is fixed by https://github.com/open-craft/dalite-ng/pull/53.